include/mappers/mapper2.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "mapper-base.h"
+
+class Mapper2 : public Mapper
+{
+
+  public:
+    Mapper2( u8 prg_rom_banks, u8 chr_rom_banks, MirrorMode mirror_mode );
+    auto TranslateCPUAddress( u16 address ) -> u32 override;
+    auto TranslatePPUAddress( u16 address ) -> u32 override;
+    void HandleCPUWrite( u16 address, u8 data ) override;
+
+    [[nodiscard]] bool SupportsPrgRam() override { return true; }
```

Comment:
As far as I understand, Mapper 2 doesn't support Prg RAM. Thoughts on disabling this?
```suggestion
    [[nodiscard]] bool SupportsPrgRam() override { return false; }
```

Reply:
You're right, that was an oversight on my part. Thanks!